misc/helm-charts/operator/values.yaml

```diff
@@ -10,56 +10,57 @@
 # Operator configuration for orchestrating Materialize
 operator:
   image:
-    # The Docker repository for the operator image
+    # -- The Docker repository for the operator image
     repository: materialize/orchestratord
-    # The tag/version of the operator image to be used
+    # -- The tag/version of the operator image to be used
     tag: v0.127.0-dev.0
-    # Policy for pulling the image: "IfNotPresent" avoids unnecessary re-pulling of images
+    # -- Policy for pulling the image: "IfNotPresent" avoids unnecessary re-pulling of images
     pullPolicy: IfNotPresent
 
   args:
-    # Log filtering settings for startup logs
+    # -- Log filtering settings for startup logs
     startupLogFilter: "INFO,mz_orchestratord=TRACE"
     enableInternalStatementLogging: true
 
   features:
-    # Flag to indicate whether to create balancerd pods for the environments
+    # -- Flag to indicate whether to create balancerd pods for the environments
     createBalancers: true
-    # Flag to indicate whether to create console pods for the environments
+    # -- Flag to indicate whether to create console pods for the environments
     createConsole: true
-    # Override the mapping of environmentd versions to console versions
+    # -- Override the mapping of environmentd versions to console versions
     consoleImageTagMapOverride: {}
-    # Whether to enable environmentd rbac checks
+    # -- Whether to enable environmentd rbac checks
     # TODO: this is not yet supported in the helm chart
     authentication: false
 
   # Cloud provider configuration
   cloudProvider:
     type: "local"
```

Comment:
Should we add `#-- Specifies cloud provider. Valid values are 'aws', 'gcp', 'azure' , 'generic', or 'local'` (or something like it?

Reply:
Thank you!